eg/demo/demo.html

```diff
@@ -51,21 +51,28 @@ <h6>Heading Level 6</h6>
 
 <h1>Font Tags</h1>
 
-Basic, unformatted font.<p>
+<p>Basic, unformatted font.</p>
 
-<font face="Comic Sans MS">Comic Sans font, with a little
- <font face="Impact">Impact</font> thrown in the middle.</font><p>
+<font face="Arial MS">Arial font, with a little
```

Comment:
Isn't the font simply called "Arial"?

The rest of the PR looks good, I'll make this fix and then release to CPAN.